lib/config/validation.ts

```diff
@@ -442,6 +442,7 @@ export async function validateConfig(
               'matchCurrentAge',
               'matchRepositories',
               'matchNewValue',
+              'matchJsonata',
```

Comment:
Would it make sense to change this hardcoded array into something like?:
```
getOptions().filter(o => o.parents?.includes('packageRules') && o.name.startsWith('match')).map(o => o.name);
```

Reply:
Yeah, we could do that now

Reply:
but out of scope for this pr

Reply:
It would have prevented the validation error on the first attempt at this change set...